Scripts/SwiftLint.sh

```diff
@@ -26,8 +26,19 @@ if [ -z "$selected_path" ]; then
 	exit 1
 fi
 
+# copy the generated output to a temp dir and strip the "swiftlint:disable:all"
 if [ "$key" = "templates_generated" ]; then
-	find "${PROJECT_DIR}/${selected_path}" -name "*.swift" -exec ${PROJECT_DIR}/Scripts/swiftlint_no_disable_all.sh {} \;
+	temp_dir=`mktemp -d -t SwiftGen`
+
+	for f in `find "${PROJECT_DIR}/${selected_path}" -name '*.swift'`; do
+		temp_file="${temp_dir}${f#*"$PROJECT_DIR"}"
```

Comment:
Not sure why you'd need the `*` here, `$f` should immediately start with `$PROJECT_DIR` since it's where we ran find, right? So no need to glob more ahead?